README.md

```diff
@@ -5,6 +5,14 @@ Github URL: https://github.com/sahat/hackathon-starter/blob/master/README.md
 
 ## Steps to run with IBM Cloud Schematics. 
 
+### Prerequisite
+
+1. A cluster which has the a public IP available and exposed to access the application. 
```

Comment:
Remove this "A cluster which has the a public IP available and exposed to access the application. " and mention it as  IBM Cloud Kubernetes cluster